src/cascadia/UnitTests_TerminalCore/ScreenSizeLimitsTest.cpp

```diff
@@ -4,64 +4,16 @@
 #include "precomp.h"
 #include <WexTestClass.h>
 
-#include "DefaultSettings.h"
 #include "../cascadia/TerminalCore/Terminal.hpp"
+#include "../cascadia/UnitTests_TerminalCore/MockTermSettings.h"

```

Comment:
since this is in the same folder as the cpp file, just `#include "MockTermStuff.h"`